### DISCUSSION TITLE
Welcome to SAS_PACKAGES Discussions!

### DISCUSSION THREAD

**Discussion by yabwon** [yabwon/SAS_PACKAGES#8] [General]
## Hi!

We’re using SAS_PACKAGES discussions on GitHub to:
  * Ask questions.
  * Share ideas.
  * Engage the community.

All the best!

**Comment by fraleu007** [comment]
Hello,

I am working in EGuide 7.15 on Windows and with SAS server 9.4 on linux.

I tried to execute the code, with downloaded SPFinit.sas in the 'SAS_Packages_yabwon'-folder:

/****/
filename packages "/sas/prd/data/FISCABA/FISCABA/Dir VI-5/TECH_Projects/SAS_Packages_yabwon";

%include packages(SPFinit.sas);
/****/

Then I get errors:

[error_log.odt](https://github.com/yabwon/SAS_PACKAGES/files/10003866/error_log.odt)

**Reply by yabwon** [reply]
Hello :-)

From what I see in the log you shared, it looks like the `SPFinint.sas` file contains some HTML/XML tags which shouldn't be there. Did you use Office Word or similar software to get the copy of the code? If so, that may be the reason.

Try to do the following steps to get proper file:

1) go to `https://github.com/yabwon/SAS_PACKAGES`
2) click on thr `Code` button:
![image](https://user-images.githubusercontent.com/9314894/201693101-e729b948-944b-43e7-bc50-9e195e6f9449.png)
3) select `Download ZIP`
![image](https://user-images.githubusercontent.com/9314894/201693470-3ff19652-5522-4e25-b6e0-774677b3169f.png)
4) save the file (`SAS_PACKAGES-main.zip`) on you Desktop 
5) extract the zip file and navigate to the `SPF` subfolder
6) copy the `SPFinit.sas` file from inside of that `SPF` folder (using WinSCP or similar software) to the directory on the server 
7) run:
```
filename packages "/sas/prd/data/FISCABA/FISCABA/Dir VI-5/TECH_Projects/SAS_Packages_yabwon";
%include packages(SPFinit.sas);
```

Looking forward your results. Let me know if it helped? 


All he best
Bart

**Reply by fraleu007** [reply]
Hi!

OK: SPFinit.sas is OK now. Thank you!

Then I tried the code:

/****/
%loadPackage(sqlinds)

data TST.test;
	set %SQL(select age from sashelp.class);

	age2 = age;
run;
/****/

This works too.

But, when I try the same code on our own DB-table with 7 million records, I get an error.

/****/
data TST.test2;
	set %SQL(select ID_DOSSIER from TAXI.DOSSIER);

	ID_DOSSIER2 = ID_DOSSIER;
run;
/****/

Is there a limit on the number of records?

[error_log2.odt](https://github.com/yabwon/SAS_PACKAGES/files/10004809/error_log2.odt)

**Reply by yabwon** [reply]
Hi,

Do you have the `sastrace=` option enabled? ([documentation]( https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/acreldb/n0732u1mr57ycrn1urf24gzo38sc.htm))

If yes, try: 
```
options sastrace=OFF;
```
and run the query one more time, and let me know how are results?

All the best
Bart

**Reply by fraleu007** [reply]
OK, it looks like this solves the problem.

Thank you!

**Reply by yabwon** [reply]
Great news! I'm glad I could help. 

If you encounter any other issues do not hesitate to start new discussion using the "New Discussion" button in the main discussions panel:
![image](https://user-images.githubusercontent.com/9314894/202138258-3e5b8e78-05d5-4ef4-ac6a-8db8080c42a5.png)


All the best
Bart
